src/test/resources/io/jenkins/plugins/folderauth/casc/expected.yml

```diff
@@ -18,32 +18,32 @@ folderRoles:
 globalRoles:
 - name: "admin"
   permissions:
-  - id: "View/Delete"
+  - id: "Agent/Build"
+  - id: "Agent/Configure"
   - id: "Agent/Connect"
```

Comment:
Reordered the expected JCasC YAML because it should now show permissions in a sorted manner.

Reply:
THANKS, I was considering making it sorted when I was working on it 😆